### DISCUSSION TITLE
Fail to install Netbox by Docker-stucking cannot proceed

### DISCUSSION THREAD

**Discussion by Greyyutech** [netbox-community/netbox-docker#592] [Getting Help]
after `docker-compose up`  it stuck and cannot proceed as below. Anyone know what is the problem ?

>netbox_1               | ✅ Initialisation is done.
netbox_1               | ⏳ Waiting for control socket to be created... (1/10)
netbox_1               | 2021/09/27 08:37:19 [warn] 6#6 Unit is running unprivileged, then it cannot use arbitrary user and group.
netbox_1               | 2021/09/27 08:37:19 [info] 6#6 unit started
netbox_1               | 2021/09/27 08:37:19 [info] 15#15 discovery started
netbox_1               | 2021/09/27 08:37:19 [notice] 15#15 module: python 3.9.5 "/usr/lib/unit/modules/python3.unit.so"
netbox_1               | 2021/09/27 08:37:19 [info] 6#6 controller started
netbox_1               | 2021/09/27 08:37:19 [notice] 6#6 process 15 exited with code 0
netbox_1               | 2021/09/27 08:37:19 [info] 17#17 router started
netbox_1               | 2021/09/27 08:37:19 [info] 17#17 OpenSSL 1.1.1l  24 Aug 2021, 101010cf
netbox_1               | 2021/09/27 08:37:19 [info] 18#18 "netbox" application started
netbox_1               | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox_1               | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox_1               | ⚙️ Applying configuration from /etc/unit/nginx-unit.json
netbox_1               | ✅ Unit configuration loaded successfully
netbox_1               | 2021/09/27 08:37:23 [notice] 6#6 process 13 exited with code 0

^CGracefully stopping... (press Ctrl+C again to force)

**Comment by tobiasge** [comment]
The logs you provided do not display any errors. Could you provide more information on what the actual problem is?

**Comment by Greyyutech** [comment]
It works when I use detached mode `docker-compose up -d`, thanks

**Comment by Maxwellchu** [comment]
I have the same issue. just stuck at there.

**Reply by cimnine** [reply]
Have you just tried to connect to NetBox?

```log
 netbox_1 | 2021/09/27 08:37:23 [notice] 6#6 process 13 exited with code 0
```

If you're worried about this, then don't. This is not an error.

**Comment by rlachman** [comment]
I am having the same issue as well, I understand it may not be an error as I can reach the NetBox webGUI without issue. But can anyone explain what it going on when it sticks on that line? Seems strange.

**Comment by tobiasge** [comment]
Nginx Unit is starting sub processes for different tasks like loading the configuration. When that specific tasks is finished the process for this tasks quits.
